bundle/config/mutator/run_as.go

```diff
@@ -183,7 +169,7 @@ func (m *setRunAs) Apply(_ context.Context, b *bundle.Bundle) diag.Diagnostics {
 
 	// Assert the run_as configuration is valid in the context of the bundle
 	if err := validateRunAs(b); err != nil {
-		return diag.FromErr(err)
+		return err
 	}
```

Comment:
You can use this pattern to only break on errors and keep accumulating warnings: https://github.com/databricks/cli/blob/2ec6abf74ebde3752467849f0d8e3093b35fed66/cmd/root/bundle.go#L83-L86

Reply:
Sure. This is old code but I'll clean it up.